src/Nethermind/Nethermind.Serialization.Rlp/RlpBehaviors.cs

```diff
@@ -30,5 +30,6 @@ public enum RlpBehaviors
     /// See https://eips.ethereum.org/EIPS/eip-4844#networking
     /// </summary>
     InMempoolForm = 64,
-    All = AllowExtraBytes | ForSealing | Storage | Eip658Receipts | AllowUnsigned | SkipTypedWrapping | InMempoolForm
+    ExcludeHashes = 128,
```

Comment:
To be honest we would love to remove RlpBehaviors all together rather than add more parameters.

Reply:
Don't see an easy change to say don't create hash or hold the data to allow to create hash later due to the plumbing of the serialization; and would be a big change to alter how that works.

Reply:
Ok, lets do it, but please think about refactoring it later...